tiledb/sm/array/array_directory.h

```diff
@@ -209,8 +224,26 @@ class ArrayDirectory {
   /** Loads the URIs from the various array subdirectories. */
   Status load();
 
-  /** Loads the fragment URIs. */
-  Status load_fragment_uris();
+  /**
+   * Loads the root directory data for v1 to v11.
+   *
+   * @return Status, vector of fragment URIs, latest fragment metadata.
+   */
+  tuple<Status, optional<std::vector<URI>>, optional<URI>> load_root_dir_data();
```

Comment:
Rename to `load_*_dir_uris` for all the load functions? "Data" may allude to loading more than URIs, e.g., fragment metadata footers.